modules/cnv-editing-vm-web.adoc

```diff
@@ -20,7 +20,6 @@ Edit select values of a virtual machine in the web console or from the *Virtual
 . You can change the *Flavor* but only to *Custom*, which provides additional fields for *CPU* and *Memory*.
 . Click *Save*.
 
-The updated values are shown after the operation is processed.
-
-
+The values and virtual machine are updated after the operation is processed.
```

Comment:
@bgaydosrh, I think you need to nest both of these statements under the final step in the procedure. I also think that you should say "... changes do not display until you reboot the virtual machine." in the second line.